penrose-web/src/ui/Canvas.tsx

```diff
@@ -45,34 +59,78 @@ class Canvas extends React.Component<ICanvasProps> {
    * @memberof Canvas
    */
   public static processData = async (data: any) => {
-    const state: State = decodeState(data);
+    // COMBAK: Clean up this use
+    const res: Either<StyErrors, State> = compileStyle();
```

Comment:
We'll definitely be using `Result` for this I think. I'll get to it after the merge.